src/driver/library_names.ml

```diff
@@ -0,0 +1,68 @@
+(** To extract the library names for a given package, without using dune, we
+
+    1. parse the META file of the package with ocamlfind to see which libraries
+    exist and what their archive name (.cma filename) is.
+
+    2. use ocamlobjinfo to get a list of all modules within the archives. EDIT:
```

Comment:
I am not 100% sure but it seems this comment is outdated, and the ocamlobjinfo step has been removed.

Reply:
I should have made it clearer, this module has been imported from https://github.com/ocaml-doc/voodoo/blob/main/src/voodoo/library_names.ml but is being used a bit differently here!